src/Form.jsx

```diff
@@ -37,7 +42,25 @@ const Form = ({ formData, setFormData }) => {
       }}
     >
       <div>
-        <p>{printColorConfig()}</p>
+        {colorConfig.map((obj, index) => (
+          <React.Fragment key={index + 1}>
+            <label>Color {(index + 1)}:</label>
+            <ChromePicker
+              color={colorConfig[index].color}
+              disableAlpha={true}
+              onChangeComplete={(color) => setColorConfigColorValue(color, index)}
+            /> 
+            <IntegerInput
+              label="Length:"
+              title="The number of stitches in this color segment"
+              name={`${index}`}
+              value={colorConfig[index].length}
+              setValue={setColorConfigLengthValue}
+              validator={IntegerInput.validators.nonNegative}
+            />
+            <br />
```

Comment:
Not a fan of using <br/> for layout unless you’re using it as a line break in text.

Reply:
I plan to remove this in later styling changes but it was good for immediate legibility

Reply:
Removed `Fragment` and made it a `div` so I could remove this `br`